scripts/alert_emails_report/generate-report.ts

```diff
@@ -70,9 +65,11 @@ async function updateSubscriptionsByLocation(subscriptions: Subscription[]) {
     return { fips, count: items.length };
   });
 
-  const [stateCounts, countyCounts] = _.partition(countByFips, ({ fips }) =>
-    isStateFips(fips),
-  );
+  const [stateCounts, countyCounts] = _.partition(countByFips, ({ fips }) => {
+    const region = regions.findByFipsCodeStrict(fips);
+    return region instanceof State;
+  });
```

Comment:
Oh, hrm.  I guess metros are probably getting counted as counties, which is probably fine for the sake of this.  You could throw a todo comment in just to help the next person that looks at this code.